lib/modules/manager/mix/__snapshots__/extract.spec.ts.snap

```diff
@@ -16,9 +16,10 @@ exports[`modules/manager/mix/extract extractPackageFile() extracts all dependenc
       "packageName": "ecto",
     },
     {
-      "currentValue": "ninenines/cowboy",
+      "currentValue": undefined,
       "datasource": "github",
       "depName": "cowboy",
+      "packageName": "cowboy",
```

Comment:
We seem to be losing the "ninenines" org in this?

Reply:
What we're losing is the GitHub repository name. This change was made following [these suggestions](https://github.com/renovatebot/renovate/pull/19128#discussion_r1035280564). This package is skipped, because we don't support renovating Elixir libraries sourced from GitHub. I don't know what's the policy when skipping a package, but given that all keys in the `PackageDependency` type are optional I would assume for skipped packages we could simply set `depName` and call it a day. Can you explain what to do in this case?

Reply:
After applying @viceice's suggestions this now changed - we're keeping the repository name in the `packageName` field if it's a `git`/`github` dependency type.